src/main/java/uk/sky/cqlmigrate/SchemaLoader.java

```diff
@@ -11,20 +11,20 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SchemaLoader.class);
 
-    private final Session session;
+    private final ExecutionInfo executionInfo;
     private final String keyspace;
     private final SchemaUpdates schemaUpdates;
     private final CqlPaths paths;
 
-    SchemaLoader(Session session, String keyspace, SchemaUpdates schemaUpdates, CqlPaths paths) {
-        this.session = session;
+    SchemaLoader(ExecutionInfo executionInfo, String keyspace, SchemaUpdates schemaUpdates, CqlPaths paths) {
+        this.executionInfo = executionInfo;
         this.keyspace = keyspace;
         this.schemaUpdates = schemaUpdates;
         this.paths = paths;
     }
 
     void load() {
-        session.execute("USE " + keyspace + ";");
+        executionInfo.getSession().execute(new SimpleStatement("USE " + keyspace + ";").setConsistencyLevel(executionInfo.getReadConsistencyLevel()));
```

Comment:
Does a USE statement *have* a consistency level?  (That's a genuine question - I don't know!)